src/reaper_osara.cpp

```diff
@@ -3713,11 +3713,18 @@ void moveToItem(int direction, bool clearSelection=true, bool select=true) {
 		if (take) {
 			s << " " << GetTakeName(take);
 		}
-		int takeCount = CountTakes(item);
-		if (takeCount > 1) {
+		int totalTakes = CountTakes(item);
+		int nonEmptyTakes = 0;
+		for (int i = 0; i < totalTakes; ++i) {
+			MediaItem_Take* take = GetTake(item, i);
+			if (take) {
```

Comment:
nit: We aren't using this variable except to check its value and the code to get the value is short, so we can simplify to: `if (GetTake(item, i))`